src/views/v2/month/calendar-body/day.php

```diff
@@ -76,7 +79,18 @@
 	>
 		<h3 class="tribe-events-calendar-month__day-date tribe-common-h6 tribe-common-h--alt">
 			<span class="tribe-common-a11y-visual-hide">
-				<?php echo esc_html( sprintf( _n( '%s event', '%s events', $day['found_events'], 'the-events-calendar' ), number_format_i18n( $day['found_events'] ) ) ); ?>,
+				<?php
+				echo esc_html(
+					sprintf(
+						/* translators: %1$s: number of events, %2$s: Event (singular), %3$s: Events (plural). */
+						_n( '%1$s %2$s', '%1$s %3$s', $day['found_events'], 'the-events-calendar' ),
```

Comment:
:warning: Mismatched placeholders is probably an error

<sub>hash: a1238f11d490a7825ead05e6b639261d</sub>